style.css

```diff
@@ -679,6 +696,146 @@ textarea {
   color: #fff;
 }
 
+.modal {
+  padding: 1rem;
+  display: none;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  z-index: 99999;
+  background-color: #dededebb;
+  backdrop-filter: blur(0.3125rem);
+  -webkit-backdrop-filter: blur(0.3125rem);
+  -moz-backdrop-filter: blur(0.3125rem);
+}
+
+.modal-wrap {
+  padding: 1rem;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  justify-content: flex-start;
+  overflow: scroll;
+}
+
+.modal-header {
+  justify-content: space-between;
+  align-items: center;
+}
+
+.modal-title {
+  font-size: 2rem;
+  font-weight: 700;
+  line-height: 2.75rem;
+  color: #172b4d;
+  text-transform: capitalize;
+}
+
+.modal-content-wrapper {
+  gap: 0.75rem;
+}
+
+.modal-content-links {
+  justify-content: space-around;
+  align-items: center;
+  gap: 1.25rem;
+  width: 100%;
+}
+
+.modal-description {
+  line-height: 1.75rem;
+  width: 100%;
+  padding: 0.7rem;
+  text-align: start;
+}
+
+.modal-content-links > a {
+  align-items: center;
+  justify-content: center;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  flex: 1;
+}
+
+.button {
+  width: -webkit-fit-content;
+  width: -moz-fit-content;
+  width: fit-content;
+  height: 3rem;
+  text-align: center;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 1.0625rem;
+  line-height: 1.5rem;
+  text-transform: capitalize;
+  border-radius: 0.5rem;
+  margin-top: 1.5rem;
+  background-color: transparent;
+}
+
+.button:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.button:focus {
+  border: 0.0625rem solid #2230d2;
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.button-enabled {
+  color: #396df2;
+  border: 0.0625rem solid #6070ff;
+}
+
+.button:disabled,
+.button[disabled] {
+  border: 0.0625rem solid #5e6c84;
+  background-color: #fff;
+  color: #5e6c84;
+}
+
+.work-categories {
+  list-style: none;
+  display: flex;
+}
```

Comment:
Great job team ✋

- [ ] [Optional] You may kindly stick to the Figma template provided as much as possible. Your may fix the colors of your technologies for the mobile version popup.
![languages](https://user-images.githubusercontent.com/63917725/180601003-af2782cb-c7cd-43a8-9e71-1cb704617059.PNG)